services/core-api/tests/factories.py

```diff
@@ -494,10 +507,11 @@ class Params:
             verified_status=None,
             mine_status=None,
             mine_tailings_storage_facilities=0,
-            mine_permit=0,
+            mine_permit_amendments=0,
```

Comment:
This seems confusing as the mine has a _mine_permit_amendments attribute but that attribute is not used for anything in the model.